pkg/network/setup/unpluggedpodnic.go

```diff
@@ -0,0 +1,85 @@
+/*
+* This file is part of the KubeVirt project
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+* Copyright 2023 Red Hat, Inc.
+*
+ */
+
+package network
+
+import (
+	"errors"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/pkg/network/namescheme"
+
+	"github.com/vishvananda/netlink"
+	k8serrors "k8s.io/apimachinery/pkg/util/errors"
+
+	netdriver "kubevirt.io/kubevirt/pkg/network/driver"
+	virtnetlink "kubevirt.io/kubevirt/pkg/network/link"
+)
+
+type Unpluggedpodnic struct {
```

Comment:
Type name formatting:
```suggestion
type UnpluggedPodNIC struct {
```

Reply:
The change seems to not apply.